src/community/label_propagation.c

```diff
@@ -337,6 +343,8 @@ igraph_error_t igraph_community_label_propagation(const igraph_t *graph,
             }
         }
         RNG_END();
+
+        control_iteration = 1 - control_iteration;
```

Comment:
Let us not rely on Boolean values begin exactly 0 and 1.

```suggestion
        control_iteration = ! control_iteration;
```